met-api/tests/utilities/factory_utils.py

```diff
@@ -876,8 +869,7 @@ def factory_enagement_content_model(
 def engagement_content_model_with_language():
     """Produce a engagement content model instance with language."""
     content_model = factory_enagement_content_model()
-    language_model = factory_language_model({'code': 'en', 'name': 'English'})
-    return content_model, language_model
+    return content_model
```

Comment:
This function is now just a wrapper for `factory_enagement_content_model()` and should be replaced with such to avoid redundancy.